src/ripple/rpc/impl/TransactionSign.cpp

```diff
@@ -706,10 +703,10 @@ Json::Value checkFee (
 
     if (fee > limit && fee != loadFee &&
         request.isMember("x-queue-okay") &&
-            request["x-queue-okay"].isBool() &&
+            request["x-queue-okay"].isConvertibleTo(Json::booleanValue) &&
```

Comment:
The feature is still experimental, so I'm experimenting. :) I was concerned that `isBool()` was too restrictive. Also, I figure that since the default is `false`, if you're including the field at all, chances are (at least for now) that your intention is to have it `true`, so I figure permissiveness is better. Further, getting into the queue increases a tx's chances of succeeding eventually; we just didn't want to make that the default.